bigxy/big_cga_test.go

```diff
@@ -59,15 +61,15 @@ func TestOrientationIndex(t *testing.T) {
 			result:       orientation.Clockwise,
 		},
 	} {
-		orientationIndex := bigxy.OrientationIndex(testData.vectorOrigin, testData.vectorEnd, testData.point)
-		if orientationIndex != testData.result {
-			t.Errorf("Test %v Failed. Expected: %v (%v) but was %v (%v) : TestData: %v", i+1, testData.result, int(testData.result), orientationIndex, int(orientationIndex), testData)
-		}
+		t.Run(fmt.Sprintf("%d", i+1), func(t *testing.T) {
```

Comment:
nit: `strconv.Itoa(x)` is nicer than `fmt.Sprintf("%d", x)`. Same applies below.